src/elements/Svg.tsx

```diff
@@ -81,16 +114,30 @@ export default class Svg extends Shape<SvgProps> {
     root && root.setNativeProps(props);
   };
 
-  toDataURL = (callback: (base64: string) => void, options?: object) => {
+  /**
+   * @deprecated use size property instead
+   * @param callback
+   * @param options as { size: { width: number, height: number }, format: 'jpeg' | 'png', quality?:1 }
```

Comment:
https://jsdoc.app/tags-callback
callback should be typed

https://jsdoc.app/tags-param

https://jsdoc.app/tags-type
quality is not an explicit `1` value but rather a number in range {0, 1}

Reply:
updated JSdoc